### PR TITLE
Removed classes to compile method

### PR DESCRIPTION
**Pull request by acrobat**
I am targeting this branch, because this fixes a deprecation warning.

## Changelog

```markdown
### Fixed
- Removed classes to compile method
```

## Subject

This fixes a deprecation warning op sf3 and php7+

### CODE REVIEW

**Comment by acrobat** [comment]
Yes you're right! Removed the whole check!

**Comment by jlamur** [comment]
@acrobat Please update the PR title and the changelog.

**Comment by acrobat** [comment]
Done @jlamur

File: src/DependencyInjection/SonataSeoExtension.php [hunk 1]
```diff
@@ -46,26 +46,12 @@ public function load(array $configs, ContainerBuilder $container)
...
-     */
-    public function configureClassesToCompile()
-    {
-        $this->addClassesToCompile([
-            'Sonata\\SeoBundle\\Seo\\SeoPage',
-            'Sonata\\SeoBundle\\Seo\\SeoPageInterface',
-            'Sonata\\SeoBundle\\Sitemap\\SourceManager',
-            'Sonata\\SeoBundle\\Twig\\Extension\\SeoExtension',
-        ]);
-    }
```
Comment (greg0ire) [inline src/DependencyInjection/SonataSeoExtension.php]: Not sur why the hell this was public but I think this BC break is acceptable

**Comment by acrobat** [comment]
I've rebased against 2.x after #226 was merged, but the classes to compile was already removed, so pr closed itself!

**Pull request closed without merging**